vipps-login-api-faq.md

```diff
@@ -298,7 +294,7 @@ Send an email to [accessuserinfo@vipps.no](mailto:accessuserinfo@vipps.no) and s
 | Links to your membership terms and privacy statement. | We must review your terms to ensure that the flow is not intended to mislead or abuse end users.                                                                                | www.merchant.com/termsandconditions <br /> www.merchant.com/privacystatement      |                                                                                                                                           |
 | Recipient to sign DPA                                 | Vipps will function as a data processor and not have any ownership to the data involved. For more information, please visit merchant terms and conditions                       | recipient@merchant.com                                                            |                                                                                                                                           |
 
-## When using delegatedConsents, which consents does Vipps support?
+## When using delegatedConsents, which consents are supported?
```

Comment:
This link breaks:
* [When using delegatedConsents, which consents does Vipps support?](#when-using-delegatedconsents-which-consents-does-vipps-support)